ccc-js/src/main/javascript/package-res/ccc/core/base/chart/chart.data.js

```diff
@@ -672,6 +672,13 @@ pvc.BaseChart
             }
         }
 
+        if ( smallDatas.length > multiOption('Max') ) {
+            chart._lastRenderMessage = {
+                code: "maxChartsReached",
+                description: "The maximum multi-chart count has been reached. To see more charts increase the number of charts in the 'Chart Options'"
```

Comment:
"Chart Options" is an Analyzer thing, which CCC does not know about. Please, either remove that statement or replace it with one that mentions the `multiChartMax` CCC option.

Reply:
Fixed